core/dbt/config/project.py

```diff
@@ -189,6 +189,12 @@ def _raw_project_from(project_root: str) -> Dict[str, Any]:
         )
 
     project_dict = _load_yaml(project_yaml_filepath)
```

Comment:
Ideally, we should just validate the complete project schema here, perhaps using something like [StrictYAML](https://hitchdev.com/strictyaml/)

Reply:
What are anchors?

Reply:
```
foo: &my_anchor
  a: 1
  b: [2, 3]
  c: false
bar: *my_anchor
```

Is equivalent to:
```
foo:
  a: 1
  b: [2, 3]
  c: false
bar:
  a: 1
  b: [2, 3]
  c: false
```

There's also a field-override syntax, which is very convenient.

Reply:
Ah yes, StrictYAML removes a lot of YAML features like that.

I'm not familiar with Hologram, but I have heard of [Cerberus](https://docs.python-cerberus.org/en/stable/) as an option for flexible data validation.